src/main/java/ch/njol/skript/expressions/ExprLoadedChunks.java

```diff
@@ -0,0 +1,79 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Chunk;
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Loaded Chunks of World")
+@Description("A list of loaded chunks of a world.")
+@Examples("set {_chunks::*} to the loaded chunks of player's world")
+@Since("INSERT VERSION")
+public class ExprLoadedChunks extends SimpleExpression<Chunk> {
+
+	static {
+		Skript.registerExpression(ExprLoadedChunks.class, Chunk.class, ExpressionType.COMBINED, "[all [of]] [the] loaded chunks (of|in) %world%");
```

Comment:
_Optional_
Can this support multiple worlds?

Reply:
At first I thought that the usage of acquiring chunks from multiple world is low. I guess it is fine to support it so users don't have to loop, unlike #5401.